azure-app-service/massdriver.yaml

```diff
@@ -204,35 +187,6 @@ params:
           title: Image Tag
           $ref: https://raw.githubusercontent.com/massdriver-cloud/artifact-definitions/main/definitions/types/docker-image-tag.json
           default: latest
-    network:
```

Comment:
The app template sets up the network automatically. If you'd like a more complicated template... fork the repo. :)